amr-wind/equation_systems/icns/source_terms/CoriolisForcing.cpp

```diff
@@ -83,9 +96,10 @@ void CoriolisForcing::operator()(
                                up[1] * vel(i, j, k, 1) +
                                up[2] * vel(i, j, k, 2);
 
-        const amrex::Real ae = +corfac * (un * sinphi - uu * cosphi);
+        const amrex::Real ae =
+            +(corfac * un * sinphi) - (corfac * uu * cosphi * S);
```

Comment:
for cosphi != 0 this will cause a diff

Reply:
I believe this is correct when S = 0 the cosine components are removed (two component forcing) when the three_component forcing flag is true S = 1, thus all the components are calculated.